packages/shared/lib/src/sdk/mod.rs

```diff
@@ -334,20 +339,20 @@ impl Sdk {
         for built_tx in built_txs.into_iter() {
             let tx_bytes = &built_tx.tx_bytes()?;
```

Comment:
Not in this PR, but is there any difference between calling `tx_bytes` then `Tx::try_from_slice` to get the Tx instead of `built_tx.tx`?

Reply:
`Tx::try_from_slice(built_tx.tx)?` should work, or `borsh::from_slice(built_tx.tx)`. I'm planning to kill this struct in a follow-up PR so I'll leave it for now.  I hate using it this way, and want to pass a simplified object around instead of this struct with methods. It's ugly to use outside of Rust at the moment. I think a better approach would have a simpler type without methods, just a group of args we need. Since I'm using serde / gloo_utils already for the vec of `signing_data` bytes, maybe that would provide a simpler interface for all of the transaction data outside of Rust, just a plain object. That's something I'm looking at next! And hopefully get rid of `wrapper_tx_msg` like you mentioned - if it happens to be needed for broadcast, the interface already has that data so in any case it is not needed here.